Paintroid/src/main/res/values/string.xml

```diff
@@ -99,6 +99,7 @@
     <string name="menu_export">Export</string>
     <string name="menu_feedback">Feedback</string>
     <string name="menu_advanced">Advanced settings</string>
+    <string name="menu_zoom_settings">Zoom Window Settings</string>
```

Comment:
For consistency sake first letter of the first word uppercase and rest lowercase: Zoom window settings